std/getopt.d

```diff
@@ -413,7 +413,7 @@ getopt(args,
 An unrecognized option such as "--baz" will be found untouched in
 $(D args) after $(D getopt) returns.
 
-$(D Help Information Generation)
+$(B Help Information Generation)
```

Comment:
I looked throughout phobos, couldn't find an example that uses H3 or H4 (just grepped for H3 and H4). I found in std.process, it simply does things like

```D
/**
   Some docs
   Command_line:
   information about command line requirements
*/
```
Which I think would work out fine. Is there something specific you had in mind?